bag_verifier/src/test/scala/uk/ac/wellcome/platform/archive/bagverifier/services/azure/AzureReplicatedBagVerifierTests.scala

```diff
@@ -0,0 +1,109 @@
+package uk.ac.wellcome.platform.archive.bagverifier.services.azure
+
+import uk.ac.wellcome.fixtures.TestWith
+import uk.ac.wellcome.platform.archive.bagverifier.services.{
+  ReplicatedBagVerifier,
+  ReplicatedBagVerifierTestCases
+}
+import uk.ac.wellcome.platform.archive.common.bagit.services.BagReader
+import uk.ac.wellcome.platform.archive.common.bagit.services.azure.AzureBagReader
+import uk.ac.wellcome.platform.archive.common.fixtures.BagBuilder
+import uk.ac.wellcome.platform.archive.common.fixtures.azure.AzureBagBuilder
+import uk.ac.wellcome.platform.archive.common.fixtures.s3.S3BagBuilder
+import uk.ac.wellcome.storage.azure.{AzureBlobLocation, AzureBlobLocationPrefix}
+import uk.ac.wellcome.storage.fixtures.AzureFixtures.Container
+import uk.ac.wellcome.storage.fixtures.S3Fixtures.Bucket
+import uk.ac.wellcome.storage.fixtures.{AzureFixtures, S3Fixtures}
+import uk.ac.wellcome.storage.s3.{S3ObjectLocation, S3ObjectLocationPrefix}
+import uk.ac.wellcome.storage.store.TypedStore
+import uk.ac.wellcome.storage.store.azure.{AzureStreamStore, AzureTypedStore}
+import uk.ac.wellcome.storage.store.s3.S3TypedStore
+import uk.ac.wellcome.storage.streaming.Codec
+import uk.ac.wellcome.storage.streaming.Codec._
+
+class AzureReplicatedBagVerifierTests
+    extends ReplicatedBagVerifierTestCases[
+      S3ObjectLocation,
+      S3ObjectLocationPrefix,
+      Bucket,
+      AzureBlobLocation,
+      AzureBlobLocationPrefix,
+      Container
+    ]
+    with S3Fixtures
+    with AzureFixtures {
+
+  val s3TypedStore = S3TypedStore[String]
+  val azureTypedStore = new AzureTypedStore[String]()(
+    implicitly[Codec[String]],
+    new AzureStreamStore()
+  )
```

Comment:
I think AzureTypedStore has the same constructor as S3TypedStore:

```suggestion
  val azureTypedStore = AzureTypedStore[String]
```

Reply:
not in the version of scala-libs we use right now. Should be added definitely, but I will focus on this PR first

Reply:
Oops, my bad. I have distinct memories of writing such a constructor; but it must have got lost in a branch somewhere.

Reply:
I updated the version of scala libs in this branch. Thanks @alexwlchan for fixing that

Reply:
You may regret doing that; said library is a bit broken, sorry: https://github.com/wellcomecollection/platform/issues/4718